src/components/Item_card/index.js

```diff
@@ -0,0 +1,32 @@
+import './Item_card.css'
+
+const Item_card = (foodItem) => {
+    return(
+        <>
+            <div className = 'Item_card'>
+                <p className='item_name'>{foodItem.foodName}</p>
+                <div className='item_details'>
+                    <p className = 'item_calories'>
+                        Calories: {foodItem.calories}
+                    </p>
+                    {foodItem.foodType ?
+                    <p className = 'item_type'>
+                        Type: {foodItem.foodType}
+                    </p> : ''
+                }
+                    {foodItem.side ?
+                    <p className = 'item_category'>
+                        side
+                    </p> :''}
+                </div>
+                <div className= 'price'>
+                    £{foodItem.price}
+                </div>
+
+
+            </div>
+        </>
+
+    )}
+
```

Comment:
Keep the empty line above export default though (sorry if this is confusing)

Reply:
😅 think i got u